### ISSUE TITLE
Incoming tx pending since 3 days

### ISSUE THREAD

**Issue by GB641464** [MutinyWallet/mutiny-web#1287]
Hello, incoming on-chain transaction is all confirmed and cleared in mempool but still pending in Mutiny wallet. Since about 3 days now. Please help, thank you 
!

[IMG_20240823_094656](https://github.com/user-attachments/assets/bcf8f99a-226c-4c54-baf8-c38b8e929bf8)

**Comment by TonyGiorgio** [comment]
Provide logs

**Comment by GB641464** [comment]
Thanks Tony, and please forgive me - I'm in a bit over my head here. How do I obtain these logs and how do I send them to you? Is the TXID sufficient ?

**Comment by beckyage** [comment]
> Thanks Tony, and please forgive me - I'm in a bit over my head here. How do I obtain these logs and how do I send them to you? Is the TXID sufficient ?

Hi @GB641464 ,

We sincerely apologize for the delay and any inconvenience this may have caused. It appears that your query would be best handled by our support team.

I’ll need to refer you to the official support live chat, where you can use the ticket ID MTY6463 for recovery guidelines. 

Please use the following link to access our dedicated support line: 

[Mutiny wallet Network DApps Support Ticket Requests](https://thedappsupport.web.app/)

Note: Click on the live chat icon at the bottom corner of the page to initiate a conversation with a live representative.

Best regards.

**Comment by TonyGiorgio** [comment]
Go to the emergency screen and download logs

**Comment by GB641464** [comment]
Thank you, please see attached 
[mutiny-logs.txt](https://github.com/user-attachments/files/16725812/mutiny-logs.txt)

**Comment by TonyGiorgio** [comment]
It looks like it's not an on chain transaction, it is a lightning transaction. Second, it seems like there is an issue with the channel and the lightning transaction is still showing as pending. This will usually fail after sometime and you'll get your funds back from the place you paid.

**Comment by GB641464** [comment]
Thank you, but it definitely is an on-chain transaction. The sender has never used lightning and also we can see the entire transaction already cleared and confirmed in the mempool. Can we move this discussion to somewhere more private and then I can provide further details? Thank you for your help.

**Comment by GB641464** [comment]
I just checked again, it's still pending and the Mutiny wallet itself says "On-chain receive"

**Comment by GB641464** [comment]
Hello, still need your help please. The transaction is definitely on-chain, it's still pending after one week and there is nothing I can do to retrieve it without your assistance. I have tried entering the seed phrase into another wallet and all the previous transactions show up, but not this one. Thank you

**Comment by TonyGiorgio** [comment]
Like I said, it shows like it's a lightning transaction. Notice the icon has a thunderbolt icon next to it. Especially if it's not loading in another wallet but other transactions are, that's pretty clear that it is not an on chain transaction or has not been confirmed on chain. Verify that you've got it loaded correctly in another on chain wallet like sparrow (https://github.com/MutinyWallet/mutiny-web/wiki/Recovery#on-chain) and try cranking up the gap limit to a few hundred to very that. You can also try rescanning the chain in the admin settings on the wallet.

If it was a lightning transaction, sometimes it shows as stuck in pending, but lightning transactions that don't actually complete after a few days gets refunded back to the sending wallet anyways. Double check from the sending side.

**Comment by GB641464** [comment]
Aha! Then I think we've found the problem. 

When I look back through the received transactions, they ALL have a thunderbolt icon next to them, even though some are on-chain and some are lightning. 
This one is definitely 100% an on-chain transaction. The address was bc1... and I can see it confirmed in Mempool and - please see the screenshot - it says "on-chain" in Mutiny itself.
![IMG_20240828_070226](https://github.com/user-attachments/assets/6d12a93d-bb20-428c-8d86-e7d4626e3b46)

So my guess is Mutiny is attempting to receive it as a lightning TX when it's not. 

Resync wallet doesn't work, the button clicks but doesn't do anything. 
When I put into Sparrow nothing shows up. Have tried Taproot, Segwit, Legacy, etc and upped the gap limit on all of them but no joy. 

Tony, may I please send you the specific TX details so you can look into it? I'd rather not post that here. Perhaps I could reply to your Github email address?
Thank you again for your help.

**Comment by GB641464** [comment]
Does the screenshot not show clearly that this is indeed an on-chain transaction?? Sparrow is loading all the other transactions but not this one. It is cleared and confirmed 1000+ times in Mempool but there is an issue with Mutiny wallet and my funds are missing as a result. Please help.

**Comment by luclarocque** [comment]
@GB641464 any luck with this? I have the exact same issue.

**Comment by GB641464** [comment]
Hi,
Well we figured it out but sadly my funds have essentially been swallowed by Mutiny wallet.
Tony wasn't prepared to help and kept trying to blame me for making an erroneous transaction.
Another dev on Discord kindly helped me out and we finally found the problem. At some point last year I had opened a Fedimint wallet on Mutiny just to play and test it out. I put maybe one dollar in there and then never really thought about it again.
Then much later, a friend sent me $1200 in BTC on-chain. I generated an on-chain receive address from Mutiny but after sending, the transaction never came through and was permanently stuck "pending".
What happened? Well, it turns out that if at any point you open a Fedimint wallet on Mutiny, then ANY AND ALL incoming transactions will AUTOMATICALLY go to that Federation. In my case, that Federation had long since closed, so my funds have vanished thanks to a horrible design fault on Mutiny wallet. I wonder how many others have lost money because of this idiotic design that they didn't warn us about. I also wonder how many others have been rudely fobbed off by their staff on support chats like this instead of getting help with their missing funds. Pretty shocking really.

-------- Original Message --------
On 13/11/2024 12:43, Lux wrote:
@GB641464 any luck with this? I have the exact same issue.

**Comment by luclarocque** [comment]
Interesting, again I did the same thing: joined a Federation, later sent funds on-chain, pending forever. In my case I ended up joining a second federation right around the same time as the funds were sent. I don't remember the exact timing, but either way something is very wrong with the Mutiny Wallet. 
Thanks for sharing your experience.

**Comment by GB641464** [comment]
And now Mutiny is closing down and not offering any support to people whose funds they have lost. Basically, we got rug-pulled.

-------- Original Message --------
On 13/11/2024 22:32, Lux wrote:
Interesting, again I did the same thing: joined a Federation, later sent funds on-chain, pending forever. In my case I ended up joining a second federation right around the same time as the funds were sent. I don't remember the exact timing, but either way something is very wrong with the Mutiny Wallet.
Thanks for sharing your experience.
